mobify/bellows
# Upgrading velocity to 1.0.0

Upgrade velocity to the version that fixes the AMD wrapper when used with r.js, and correct vendor prefixing with flex.

Status: **Ready to merge**

Reviewers: @kpeatt @tedtate 
## Changes
- upgraded velocity to 1.0.0
## How to test-drive this PR
- checkout this branch
- `grunt build-dist && grunt test`
- `grunt serve` and test AMD and non-AMD versions in the examples
- integrate into Adaptive ~1.0.0 project and ensure bellows can be required in correctly (I tested this myself)
